locale/ru/docs/guides/debugging-getting-started.md

```diff
@@ -0,0 +1,244 @@
+code on behalf of the Node process. It is important to understand the security
+implications of exposing the debugger port on public and private networks.
+
+### Exposing the debug port publicly is unsafe
+
+If the debugger is bound to a public IP address, or to 0.0.0.0, any clients that
+can reach your IP address will be able to connect to the debugger without any
+restriction and will be able to run arbitrary code.
+
+By default `node --inspect` binds to 127.0.0.1. You explicitly need to provide a
+public IP address or 0.0.0.0, etc., if you intend to allow external connections
+to the debugger. Doing so may expose you a potentially significant security
+threat. We suggest you ensure appropriate firewalls and access controls in place
+to prevent a security exposure.
+
+See the section on '[Enabling remote debugging scenarios](#enabling-remote-debugging-scenarios)' on some advice on how
+to safely allow remote debugger clients to connect.
+
+### Local applications have full access to the inspector
+
+Even if you bind the inspector port to 127.0.0.1 (the default), any applications
+running locally on your machine will have unrestricted access. This is by design
+to allow local debuggers to be able to attach conveniently.
+
+### Browsers, WebSockets and same-origin policy
+
+Websites open in a web-browser can make WebSocket and HTTP requests under the
+browser security model. An initial HTTP connection is necessary to obtain a
+unique debugger session id. The same-origin-policy prevents websites from being
+able to make this HTTP connection. For additional security against
+[DNS rebinding attacks](https://en.wikipedia.org/wiki/DNS_rebinding), Node.js
+verifies that the 'Host' headers for the connection either
+specify an IP address or `localhost` or `localhost6` precisely.
+
+These security policies disallow connecting to a remote debug server by
+specifying the hostname. You can work-around this restriction by specifying
+either the IP address or by using ssh tunnels as described below.
+
+## Inspector Clients
+
+Several commercial and open source tools can connect to Node's Inspector. Basic
+info on these follows:
+
+#### [node-inspect](https://github.com/nodejs/node-inspect)
+
+* CLI Debugger supported by the Node.js Foundation which uses the [Inspector Protocol][].
+* A version is bundled with Node and can be used with `node inspect myscript.js`.
+* The latest version can also be installed independently (e.g. `npm install -g node-inspect`)
+  and used with `node-inspect myscript.js`.
+
+#### [Chrome DevTools](https://github.com/ChromeDevTools/devtools-frontend) 55+
+
+* **Option 1**: Open `chrome://inspect` in a Chromium-based
+  browser. Click the Configure button and ensure your target host and port
+  are listed.
+* **Option 2**: Copy the `devtoolsFrontendUrl` from the output of `/json/list`
+  (see above) or the --inspect hint text and paste into Chrome.
+* **Option 3**: Install the Chrome Extension NIM (Node Inspector Manager):  
```

Comment:
This should be changed once/if https://github.com/nodejs/nodejs.org/pull/2259 gets landed.

Reply:
@alexandrtovmach ：Please change this because #2259 gets landed, according to what @ChALkeR 's suggestion :)

Reply:
Okay, done